Mathlib/MeasureTheory/Function/AEEqFun.lean

```diff
@@ -859,6 +859,12 @@ instance : Star (α →ₘ[μ] R) where
 lemma coeFn_star (f : α →ₘ[μ] R) : ↑(star f) =ᵐ[μ] (star f : α → R) :=
    coeFn_comp _ (continuous_star) f
 
+instance : InvolutiveStar (α →ₘ[μ] R) where
+  star_involutive f := by
+    ext
+    filter_upwards [AEEqFun.coeFn_star (star f), AEEqFun.coeFn_star f] with x hx hy
+    simp only [hx, Pi.star_apply, hy, star_star]
```

Comment:
You can golf this a little by proving
```lean-4
@[simp]
theorem comp_id (f : α →ₘ[μ] β) : comp id (continuous_id) f = f := by
  rcases f; rfl

@[simp]
theorem comp_comp (g : γ → δ) (g' : β → γ) (hg : Continuous g) (hg' : Continuous g') (f : α →ₘ[μ] β) :
    comp g hg (comp g' hg' f) = comp (g ∘ g') (hg.comp hg') f := by
  rcases f; rfl
```
higher up the file, and then replacing this proof with
```lean-4
star_involutive f := comp_comp _ _ _ _ f |>.trans <| by simp [star_involutive.comp_self]
```

Reply:
Very cool!